alertmanager/version.txt

```diff
@@ -1 +1 @@
-0.0.0
+0.11.1
```

Comment:
I think you mentioned before that you want to start from a version greater than any existing version and just move forward from there. So I updated all vendor images to 0.11.1

Reply:
now bumped to 0.11.2